src/integrationTest/resources/reconciliation/invalid-supplier-statement-report.json

```diff
@@ -1,7 +1,6 @@
 {
-  "envelopes": {
-    "zip_file_name": "2010404021234_14-08-2020-03-06-18.zip",
-    "container": "bulkscan",
-    "jurisdiction": "BULKSCAN"
+  "report": {
```

Comment:
not related to this PR, but is this field in the spec??

Reply:
I remember copying it from the samples in reconciliation design proposals, but I don't see it now. I'll raise a pr to remove `report`.

Reply:
Fixed the request model to match with the spec